Game/level.gd

```diff
@@ -59,14 +62,6 @@ func _on_quit_button_pressed():
 	get_node("./QuitMenu/QuitWindow").show()
 	get_tree().paused = true
 
-func _on_no_button_pressed():
-	get_node("./QuitMenu/QuitWindow").hide()
-	get_tree().paused = false
-
-func _on_back_button_pressed():
-	get_tree().paused = false
-	go_back()
-	
 func go_back():
 	get_tree().change_scene_to_file("res://start_menu.tscn")
 
```

Comment:
Don't delete _on_back_button_pressed since that is what the quit button calls.  Combine these two functions.

Reply:
oh whoops, i think that was an accident

Reply:
also that function is called other times in the file so idk if i should combine them

Reply:
Then just leave them separate.

Reply:
i am going to do that